config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml

```diff
@@ -7,7 +7,7 @@ presubmits:
     path_alias: "sigs.k8s.io/cluster-api-provider-aws"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-1.13
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-1.13
```

Comment:
rotation missing (in the entire script). This job should move to 1.14

Reply:
this is presubmit for other repos, we should avoid touching their versions, aka a go version bump might break their repo